app/models/version.rb

```diff
@@ -332,6 +332,10 @@ def yanker
     Deletion.find_by(rubygem: rubygem.name, number: number, platform: platform)&.user unless indexed
   end
 
+  def prerelease
+    self[:prerelease] ||= !!to_gem_version.prerelease?
```

Comment:
:+1: Nice refactoring! 

:thinking: I'm just wondering if we actually need the memoization in here. `to_gem_version` is not memoized and depends on `number` column. Anyway similar problem is with old implementation (it can get stale values as well). I can take a look at this and make it safer outside of this PR.